src/Ydb.Sdk/tests/Auth/TestStaticAuth.cs

```diff
@@ -0,0 +1,110 @@
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Xunit;
+using Xunit.Abstractions;
+using Ydb.Sdk.Auth;
+using Ydb.Sdk.Services.Table;
+
+namespace Ydb.Sdk.Tests.Auth;
+
+[Trait("Category", "Integration")]
+public class TestStaticAuth
+{
+    // ReSharper disable once NotAccessedField.Local
+    private readonly ITestOutputHelper _output;
+    private readonly ILoggerFactory? _loggerFactory;
+    private readonly ILogger _logger;
+
+    public TestStaticAuth(ITestOutputHelper output)
+    {
+        _output = output;
+        _loggerFactory = Utils.GetLoggerFactory() ?? NullLoggerFactory.Instance;
+        _logger = _loggerFactory.CreateLogger<TestStaticAuth>();
+    }
+
+    private async Task DoAuth(string? passwordCreate, string? passwordAuth, int maxRetries = 5)
```

Comment:
Is .net provide setup/teardown methods - for create user once at tests startss and remove at test finished?

If yes - better to inialize user and remove it on class level and sparate it from try of auth.